readthedocs/projects/version_handling.py

```diff
@@ -36,7 +36,7 @@ def prune_major(self, num_latest):
         all_keys = sorted(set(self._state.keys()))
         major_keep = []
         for __ in range(num_latest):
-            if len(all_keys) > 0:
+            if all_keys:
```

Comment:
Seems like the previous approach was more explicit (eg. it must be a list, and will raise an exception if `None` is there). Not sure if we care, but it does seem like this is changing logic some.